src/scanner/main.rs

```diff
@@ -0,0 +1,476 @@
+extern crate shell_words;
```

Comment:
It's annoying that this entire file is classes as "new" but I don't know how to fix it :cry:

Reply:
I can fix it for you, no worries! :)

Reply:
Your local master is probably out of sync with the remote master.